.gitattributes

```diff
@@ -1 +1,2 @@
-.licenses/** -diff linguist-generated=true
\ No newline at end of file
+.licenses/** -diff linguist-generated=true
+* text=auto eol=lf
```

Comment:
windows-latest build was failing. To fix added this. See pt 4 at https://prettier.io/docs/en/options.html#end-of-line